tests/features/eulogin/eulogin.feature

```diff
@@ -102,6 +121,8 @@ Feature: Log in through EU Login
     And users:
       | Username             | Password | E-mail                           | First name | Family name | Organisation |
       | chuck_the_local_hero | 12345    | chuck_the_local_hero@example.com | LocalChick | LocalNorris | ACME         |
+    # Test that the user is not redirected anymore to accept the 'Legal notice'.
+    And User "chuck_the_local_hero" accepts the "Legal notice" agreement
```

Comment:
This is a step used to set up the test environment so it is better to not word it like an action taken by the user, but to refer to it in a passive form. The idea of "@Given" is to describe the condition of the test environment.

Suggestion:

`And the "Legal notice" agreement has been accepted by the "chuck_the_local_hero" user`